task-features/pablo/attachments.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Trello API attachments
+    As a user
+    I want "Attachments" API endpoints
+    In order to manipulate attachments
+
+    Scenario: GET all attachments from a card
+        Given there is a card with attachments
```

Comment:
it is ok, however I think it would be better if in the Given we specify that the card was created, with "there is" we state that there will always be a card when in fact we may have to create it in case there is not one by default... i mean, if someone deletes the card then we would have to create it in this first step
```suggestion
        Given It was created a card with attachments
```